app.py

```diff
@@ -129,11 +138,13 @@ def execute():
     success = False
     req = request.get_json()
     try:
-        response = redis_db.execute_command(*req['command'].split())
+        conn = get_conn_through_sentinel()
```

Comment:
shouldn't this replace redis_db above instead?
this has the additional distinction of creating a conn per execute request, while previously - intentionally or not - the connection was setup only once

Reply:
You mentioned to me a test case that master changes. On such a case sentinel will return a different master for each time, and it might be better to refresh. If this is cool I should probably remove the unused global variable, good point

Reply:
It's a good point, but somehow the code was already handling that